api/server/server.go

```diff
@@ -847,12 +847,28 @@ func postContainersRestart(eng *engine.Engine, version version.Version, w http.R
 	if vars == nil {
 		return fmt.Errorf("Missing parameter")
 	}
-	job := eng.Job("restart", vars["name"])
-	job.Setenv("t", r.Form.Get("t"))
-	if err := job.Run(); err != nil {
+
+	seconds := 10
+	t := r.Form.Get("t")
+	if t != "" {
+		s, err := strconv.Atoi(t)
+		if err != nil {
+			return err
+		}
+		seconds = s
+	}
+	//s, err := strconv.Atoi(t)
```

Comment:
I'd prefer if the default (10) was pushed down into ContainerRestart() and not set here at the API level - otherwise we run the risk of having different defaults based on how ContainerRestart() is called.  Can we have ContainerRestart() treat 0 as "no value specified" and then have it default to 10?